app/controllers/users_controller.rb

```diff
@@ -38,6 +52,10 @@ def user_params
     params.require(:user).permit(:password, :password_confirmation)
```

Comment:
Include `:current_password` (take a look at the view). Then make sure that the controller really checks for current password before changing password.